frontend/src/components/Pages/Todo/todo.css

```diff
@@ -1,5 +1,33 @@
 @import "../../../styles/variables.css";
 
+.checked-style {
+  color: #e6edf4;
+  text-decoration: line-through;
+}
+.unchecked-style {
+  color: white;
+}
+
+.list-row {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.group-list-icon {
+  margin-right: 10px;
+}
+
+.group-list-icon.completed {
+  color: #FBEBA5;
```

Comment:
Change the hex colour code to have lowercase letters -- to keep it consistent with the rest of the CSS files.